compositor_common/src/scene/common_transformations.rs

```diff
@@ -0,0 +1,22 @@
+use serde::{Deserialize, Serialize};
+
+use crate::util::RGBAColor;
+
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(tag = "type", content = "params", rename_all = "snake_case")]
+pub enum CommonTransformation {
+    ConvertResolution(ConvertResolutionParams),
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(tag = "strategy", content = "color", rename_all = "snake_case")]
+pub enum ConvertResolutionParams {
+    /// Rescales input in both axis to match output resolution
+    Stretch,
+    /// Scales input preserving aspect ratio and cuts equal parts
+    /// from both sides in "sticking out" dimension
+    CropScale,
+    /// Scales input preserving aspect ratio and
+    /// fill the rest of the texture with the provided color
+    FillScale(RGBAColor),
+}
```

Comment:
Not sure about this naming. Any better ideas?

Reply:
FillScale - this is not correct, fill in this context usually means that the image would fill the entire view which would be the same as Crop

I would name it just Fill(instead of CropScale) and Fit(instead of FillScale). The scale does not provide any info.